soaks/Dockerfile

```diff
@@ -2,19 +2,21 @@ ARG RUST_VERSION=1.56
 ARG DEBIAN_VERSION=bullseye
 
 #
-# BUILDER
+# VECTOR BUILDER
 #
 FROM docker.io/rust:${RUST_VERSION}-${DEBIAN_VERSION} as builder
-RUN apt-get -y update && apt-get -y install build-essential cmake libclang-dev libsasl2-dev
+RUN apt-get update && apt-get -y install build-essential git clang cmake libclang-dev libsasl2-dev libstdc++-10-dev libssl-dev libxxhash-dev zlib1g-dev zlib1g
+RUN git clone https://github.com/rui314/mold.git
```

Comment:
Should we use the same linker we use for release builds? Or do you not anticipate differences in behavior. Alternatively we could use `mold` there too.

Reply:
Different goals. The soak tests compare with themselves and variation between them and the actual release build is expected. 

We can, in the future, make soaks with the nightly release containers but that's a slightly different ambition.